tables/ru-unicode.dis

```diff
@@ -0,0 +1,107 @@
+#  Copyright (C) 2018 Bert Frees <bertfrees@gmail.com>
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
```

Comment:
Ugh, we want LGPL v2.1+. Just s/3/2.1/

Reply:
Sure, I just copy pasted it from some other table.